core/README.md

```diff
@@ -33,3 +40,13 @@ The process of adding new collaterals depends on the token type used. This is du
         - The file should be named using the name from `1.`
         - The file should export `CalleeFunctions`
         - The file should be imported in the [`calleeFunctions/index.ts`](./src/calleeFunctions/index.ts)
+3. Adding price oracle configurations for the token:
+    1. Get the source code of the price oracle contract:
+       - read `ilks(collateralType)` from [`Spot` contract](https://etherscan.io/address/0x65c79fcb50ca1594b025960e539ed7a9a6d434a3#code)
+    2. Read the contract and determine the slot address of the variable:
+       - Depending on the variable type it can be stored in different slots for memory consumption minimization
+       - https://docs.soliditylang.org/en/v0.8.13/internals/layout_in_storage.html documentation is helpful
+       - Generally the slot number increments with each (big enough) variable in top-to-bottom manner
+       - Experimenting with blockchain fork (e.g. hardhat) helps: try to fetch the value you're looking for / overwrite it / ... and validate that it's correct via some public method or comparing against your expectation.
```

Comment:
This doesn't explain how to actually experiment with it.

Currently, even me with internal knowledge of the repo, contracts and the process, wouldn't be able to add a new collateral because this instruction doesn't explain it practical enough. Where to look? What commands to execute? How to understand they are correctly executed? What `private variables` need to be extracted? What to do with the slot address after it was determined? What is the difference between `CurrentAndNextPrice` and the other type? What is `hasDelay` value and how to determine it? Where is the example of the config that needs to be added? What is `offset`? What is `slotPriceValueBeginsAtPosition`? etc, I can keep going with every other value.

Please explain it in details that even external developer can add a new collateral / fix existing one, provide links to the source code. Provide helper functions/scripts to determine slots if this will simplify the process

Reply:
added a section